hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/recovery/ReconOmMetadataManagerImpl.java

```diff
@@ -108,4 +109,14 @@ public void updateOmDB(File newDbLocation) throws IOException {
     initializeNewRdbStore(newDbLocation);
   }
 
+  @Override
+  public long getLastSequenceNumberFromOMMetadataDB () {
```

Comment:
Can be renamed to 'getLastSequenceNumberFromDB' since the method is now in OM Metadata Manager.